templates/committees-single.html

```diff
@@ -10,7 +10,11 @@
       <div class="container">
         <div class="entity__header--top row">
             <h1 class="entity__name">{{ name }}</h1>
-            <span class="entity__type"><img class="category-icon" width="20" height="20" src="/static/img/icon-committee--white.svg" alt="Icon representing committees">Committee - {{ designation }}</span>
+            <span class="entity__type">
+              <img class="category-icon"
+                   width="20" height="20"
+                   src="/static/img/icon-committee--white.svg"
+                   alt="Icon representing committees">Committee - {{ designation }}</span>
```

Comment:
My change here was actually just putting things on newlines because it was a really wide line. I'm  confused what you mean by `{{ designation }} - {{ type }} - {{ organization }}`.